build.c

```diff
@@ -259,9 +261,12 @@ jobstart(struct job *j, struct edge *e)
 	struct node *n;
 	struct string *rspfile, *content, *description;
 	int fd[2];
+	unsigned int len;
```

Comment:
This should be `size_t`.

Reply:
But since it's use is for "%.*s" with printf I'd need to cast it to int anyway, should I change it to size_t and just add a cast anyway?

Reply:
Yes, I think so. That is an implementation detail of `printdesc`, and should not leak into its interface.

Reply:
I'd also call it `statuslen` to make it clear what it's a length of.